packages/expo/package.json

```diff
@@ -100,9 +108,6 @@
     },
     "expo-local-authentication": {
       "optional": true
-    },
-    "expo-secure-store": {
-      "optional": true
```

Comment:
We should remove this in the next major, because if someone is not using this package they will get errors when updating their package.

Reply:
Good catch and I agree, thanks!